### PR TITLE
add /landing route & dump header component

### PR DESCRIPTION
**Pull request by msharafjr**

### CODE REVIEW

File: src/app/components/header.tsx [hunk 1]
```diff
@@ -0,0 +1,34 @@
+import React from "react";
+import classnames from "classnames-prefix";
+import {NavMenu} from './nav-menu';
+import {SocilLinks} from './social-links';
+import {FeedLink} from './feed-link';
+import {Sidebar} from './sidebar';
```
Comment (shadyvb) [inline src/app/components/header.tsx]: Components should be exported as defaults as a general rule.

File: src/app/components/header.tsx [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+
+const cx = classnames('header');
+
+
+const Logo = () => (
+  <a className={cx('__logo')} href='/'>
```
Comment (shadyvb) [inline src/app/components/header.tsx]: This should also be a `Link` component.

File: src/app/components/header.tsx [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+import classnames from "classnames-prefix";
+import {NavMenu} from './nav-menu';
+import {SocilLinks} from './social-links';
+import {FeedLink} from './feed-link';
+import {Sidebar} from './sidebar';
+
+const cx = classnames('header');
+
+
+const Logo = () => (
```
Comment (shadyvb) [inline src/app/components/header.tsx]: One component per file.

File: src/app/routes/index.ts [hunk 4]
```diff
@@ -9,6 +10,6 @@ type route = {
   };
 };
 
-const routes: route[] = [home, catchall];
+const routes: route[] = [home, landing, catchall];
```
Comment (shadyvb) [inline src/app/routes/index.ts]: Replace `landing` with `home`, we don't need the existing home anymore.

**Review by shadyvb — CHANGES_REQUESTED** [review]
.

File: src/app/components/social-links.tsx [hunk 5]
```diff
@@ -0,0 +1,28 @@
...
+const links = [
+  {icon: 'F', path: 'https://facebook.com'},
+  {icon: 'T', path: 'https://twitter.com'},
+  {icon: 'Y', path: 'https://youtube.com'},
+  {icon: 'I', path: 'https://instagram.com'},
+  {icon: 'F', path: 'https://flipboard.com'},
+  {icon: 'F', path: '/feed'},
+];
+
+const SocilLinks = () => (
```
Comment (shadyvb) [inline src/app/components/social-links.tsx]:
```suggestion
const SocialLinks = () => (
```

File: src/app/components/sidebar.tsx [hunk 6]
```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+import classnames from "classnames-prefix";
+
+const cx = classnames('header');
+
+const Sidebar = () => (
+  <div className={cx('__side-nav')}>+</div>
```
Comment (shadyvb) [inline src/app/components/sidebar.tsx]: Class names should match the component name as much as possible.

File: src/app/components/nav-menu.tsx [hunk 7]
```diff
@@ -0,0 +1,38 @@
...
+  {title: 'News', path: '/news'},
+  {title: 'Exclosive', path: '/exclosive'},
+  {title: 'Movies', path: '/movies'},
+  {title: 'Tv', path: '/tv'},
+  {title: 'Gaming', path: '/gaming'},
+];
+
+const cx = classnames('header');
+
+const NavLink = (props: any) => {
```
Comment (shadyvb) [inline src/app/components/nav-menu.tsx]: Duplicate component.

File: src/app/routes/counter.tsx [hunk 8]
```diff
@@ -0,0 +1,65 @@
+import React from "react";
```
Comment (shadyvb) [inline src/app/routes/counter.tsx:1]: Need to remove this route.

**Review by shadyvb — CHANGES_REQUESTED** [review]
.

**Comment by msharafjr** [comment]
@shadyvb

**Review by shadyvb — APPROVED** [review]
Okay, I give up, we'll fix it later 😒

**Pull request merged**